stemcell_builder/stages/bosh_go_agent/assets/nats-access-helper.sh

```diff
@@ -0,0 +1,23 @@
+# This is the integer value of the argument "0xb0540001", which is
+# b054:0001 . The major number (the left-hand side) is "BOSH", leet-ified.
+# The minor number (the right-hand side) is 1, indicating that this is the
+# first thing in our "BOSH" classid namespace.
```

Comment:
It looks like the minor number of the classid in this file is two. Two things come to mind:

1) This comment should be updated appropriately.
2) I think that these classids should be moved into a single file that contains descriptions of what they control, so that we can better understand what classids are in use... rather than having to search through all shell scripts in this repo and hoping that we find the right stuff.

Reply:
updated